hips/draw/simple.py

```diff
@@ -83,111 +87,60 @@ def dst(self) -> np.ndarray:
              [0, 0]],
         )
 
-    @property
-    def projection(self) -> ProjectiveTransform:
+    def projection(self, tile: HipsTile) -> ProjectiveTransform:
         """Estimate projective transformation on a HiPS tile"""
-        corners = self.tile.meta.skycoord_corners.to_pixel(self.geometry.wcs)
+        corners = tile.meta.skycoord_corners.to_pixel(self.geometry.wcs)
         src = np.array(corners).T.reshape((4, 2))
         dst = self.dst
         pt = ProjectiveTransform()
         pt.estimate(src, dst)
         return pt
 
-    def warp_image(self) -> np.ndarray:
+    def fetch_tiles(self) -> 'HipsTile':
+        """Fetch HiPS tiles from a remote URL"""
+        for healpix_pixel_index in self.tile_indices:
+            tile_meta = HipsTileMeta(
+                order=self.hips_order,
+                ipix=healpix_pixel_index,
+                frame=self.hips_survey.astropy_frame,
+                file_format=self.tile_format,
+            )
+            tile = HipsTile.fetch(tile_meta, self.hips_survey.tile_access_url(order=self.hips_order, ipix=healpix_pixel_index) + tile_meta.filename)
+            yield tile
+
+    @property
+    def shape(self) -> tuple:
```

Comment:
Suggest to rename to more explicit `output_image_shape`, and to make a more precise type annotation (I think it's `Tuple[int]`

The description below is not quite correct. You say "four extra dimensions".

What we really have is three dimensions for RGB images, and there are four "channels" for the case of PNG:
```
In [1]: import numpy as np

In [2]: data = np.zeros((100, 200, 4))

In [3]: data.shape
Out[3]: (100, 200, 4)

In [4]: data.ndim
Out[4]: 3
```

My suggestion here would be that you state that we follow the same axis order and coordinate conventions that are used by others for grayscale and RGB images:
http://scikit-image.org/docs/stable/user_guide/numpy_images.html#coordinate-conventions
and then give examples (like they do on that page, repeating the info relevant for uw) showing the shape.